AbletonMCP_Remote_Script/handlers/browser.py

```
@@ -0,0 +1,653 @@
+        if not app:
+            raise RuntimeError("Could not access Live application")
+        browser = app.browser
+        if action == "stop":
+            browser.stop_preview()
+            return {"action": "stop", "previewing": False}
+        elif action == "preview":
+            if not uri:
+                raise ValueError("uri is required for preview action")
+            item = find_browser_item_by_uri(browser, uri, ctrl=ctrl)
+            if item is None:
+                raise Exception("Browser item not found for URI: {0}".format(uri))
```

Ruff (0.15.0)

[warning] Abstract `raise` to an inner function

(TRY301)